### PR TITLE
Make replies appear in retags

### LINKED ISSUE #326: make retags show replies
>Hi! First of all thanks for doing what you do! New XKit is awesome and so are all of you! Okay to keep this brief - when using "retags" to show tags on notes, there's an option for the activity graph to only show retags/responses. It's a toggleable option on the activity page. It makes it so only notes with reblog comments or tags appear on the activity page. Can you make it so it also shows replies that people leave using the reply button? Currently it hides those.

seems legit. thoughts?

### PR DESCRIPTION
**Pull request by blackjackkent**
Fixes #326

### CODE REVIEW

**Comment by bvtsang** [comment]
Don't forget to bump up the version.

**Comment by blackjackkent** [comment]
Done.

I wrote a whole checklist hook for myself for this...and it's on my computer at home. :P 

Karma punishing me for doing OSS at work. ;)

**Comment by finagle29** [comment]
tested locally, didn't work until I changed `is_reply` to `is_answer`

**Comment by blackjackkent** [comment]
Okay, that's really baffling. :/ I can't find that class anywhere in the activity page code.

reply

reply_source

Posting screenshots here...can you confirm we're looking at the same thing?

**Comment by finagle29** [comment]
that's really weird, maybe do it for both `is_reply` and `is_answer`?  On my activity page `$(".is_reply")` returns null. Maybe randomly choosing whether or not replies are replies or answers is a new tumblr feature???

screen shot 2015-08-22 at 3 56 16 pm

**Comment by nightpool** [comment]
@finagle29 replies are answers if you have a question mark at the end of
your post. otherwise they're just replies.

**Comment by rosalindwills** [comment]
Ohhhh.

I guess that makes sense. Sort of. In a weird tumblr-ish way. I'll add both in.

**Comment by finagle29** [comment]
thanks @nightpool for clearing this up

**Comment by rosalindwills** [comment]
Updated.

**Comment by blackjackkent** [comment]
Updated.

File: Extensions/retags.js [hunk 1]
```diff
@@ -70,6 +70,10 @@ XKit.extensions.retags = {
 				$c = $t.find('.notification_sentence');
 				url = $c.find('.notification_target').attr('href');
 			}
+			//we don't need to put tags on a reply, but we also don't need to hide it
+			if ($t.hasClass('is_reply')) {
```
Comment (finagle29) [inline Extensions/retags.js]: probably should check for `is_answer` here too

**Comment by blackjackkent** [comment]
Good call. Updated.

**Comment by finagle29** [comment]
tested locally, LGTM
@homu r+

**Pull request merged**